tests/test_unit/test_cdf_tk/test_utils/test_http_client.py

```diff
@@ -204,6 +204,22 @@ def test_error_text(self, http_client: HTTPClient, rsps: responses.RequestsMock)
         assert response.status_code == 401
         assert response.error == '{"message": "plain_text"}'
 
+    def test_request_alpha(self, http_client: HTTPClient, rsps: responses.RequestsMock) -> None:
```

Comment:
You could have had a test that you are not always sending alpha by mistake, but I think that is something you can pick up as part of a possible future fix. Nothing to block this PR for.